schedule/qam/common/mau-extratests-phub.yaml

```diff
@@ -16,14 +16,39 @@ conditional_schedule:
     VERSION:
       15-SP3:
         - '{{sle15_x86_64}}'
+        - '{{sle15_aarch64}}'
+        - '{{sle15_ppc64le}}'
+        - '{{sle15_s390x}}'
       15-SP2:
         - '{{sle15_x86_64}}'
+        - '{{sle15_aarch64}}'
+        - '{{sle15_ppc64le}}'
+        - '{{sle15_s390x}}'
       15-SP1:
         - '{{sle15_x86_64}}'
+        - '{{sle15_aarch64}}'
+        - '{{sle15_ppc64le}}'
+        - '{{sle15_s390x}}'
       15:
         - '{{sle15_x86_64}}'
+        - '{{sle15_aarch64}}'
+        - '{{sle15_ppc64le}}'
+        - '{{sle15_s390x}}'
   sle15_x86_64:
     ARCH:
       x86_64:
         - console/wpa_supplicant
+        - console/python_flake8
+  sle15_aarch64:
```

Comment:
Why don't you place `console/python_flake8` somewhere as any other tests e.g. `console/vmstat`, instead of this unneeded splitting ?

Reply:
`wpa_supplicant` does have this condition because it's running only on s86_64

Reply:
clarified

Reply:
@dzedro , Thank you for suggesting the correct way of scheduling this test for >SLE15 releases.